tests/test_backend.py

```diff
@@ -824,6 +824,35 @@ def test_run_not_supported_archive(self):
             self.assertEqual(item['uuid'], expected_uuid)
             self.assertEqual(item['tag'], 'test')
 
+    def test_run_json_line(self):
+        """Test run method with --json-line"""
+
+        args = ['-u', 'jsmith', '-p', '1234', '-t', 'abcd',
+                '--archive-path', self.test_path,
+                '--from-date', '2015-01-01', '--tag', 'test',
+                '--output', self.fout_path, 'http://example.com/',
+                '--json-line']
+
+        cmd = MockedBackendCommand(*args)
+        cmd.run()
+        cmd.outfile.close()
+
+#        items = [item for item in convert_cmd_output_to_json(self.fout_path)]
```

Comment:
Can you remove this line before merging it?

Reply:
Done.